packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2612,6 +2647,7 @@ class SemanticsConfiguration {
       .._value = _value
```

Comment:
I can't comment on the correct line, but `routeName` probably also needs to be handled in `absorb` and `isCompatibleWith` above.

Also, is it ok for the `routeName` to implicitly get merged into other semantics nodes? Or should a node that specifies a routeName always be a semantics boundary meaning that semantic information from that route cannot bubble up to parent nodes of that route?